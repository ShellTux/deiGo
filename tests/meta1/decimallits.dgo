0. 12. .3 .456 7.8 9.01 234.567 0.e1 12.e12 .3e1 .456e498 7.8e35 9.01e7985 234.567e28 0.e+1 12.e+12 .3e+1 .456e+498 7.8e+35 9.01e+7985 234.567e+28 0.e-1 12.e-12 .3e-1 .456e-498 7.8e-35 9.01e-7985 234.567e-28 0.E1 12.E12 .3E1 .456E498 7.8E35 9.01E7985 234.567E28 0.E+1 12.E+12 .3E+1 .456E+498 7.8E+35 9.01E+7985 234.567E+28 0.E-1 12.E-12 .3E-1 .456E-498 7.8E-35 9.01E-7985 234.567E-28 1E 9e 2.e 3.E .e .E
