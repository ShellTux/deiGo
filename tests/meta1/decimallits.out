DECIMAL(0.)
DECIMAL(12.)
DECIMAL(.3)
DECIMAL(.456)
DECIMAL(7.8)
DECIMAL(9.01)
DECIMAL(234.567)
DECIMAL(0.e1)
DECIMAL(12.e12)
DECIMAL(.3e1)
DECIMAL(.456e498)
DECIMAL(7.8e35)
DECIMAL(9.01e7985)
DECIMAL(234.567e28)
DECIMAL(0.e+1)
DECIMAL(12.e+12)
DECIMAL(.3e+1)
DECIMAL(.456e+498)
DECIMAL(7.8e+35)
DECIMAL(9.01e+7985)
DECIMAL(234.567e+28)
DECIMAL(0.e-1)
DECIMAL(12.e-12)
DECIMAL(.3e-1)
DECIMAL(.456e-498)
DECIMAL(7.8e-35)
DECIMAL(9.01e-7985)
DECIMAL(234.567e-28)
DECIMAL(0.E1)
DECIMAL(12.E12)
DECIMAL(.3E1)
DECIMAL(.456E498)
DECIMAL(7.8E35)
DECIMAL(9.01E7985)
DECIMAL(234.567E28)
DECIMAL(0.E+1)
DECIMAL(12.E+12)
DECIMAL(.3E+1)
DECIMAL(.456E+498)
DECIMAL(7.8E+35)
DECIMAL(9.01E+7985)
DECIMAL(234.567E+28)
DECIMAL(0.E-1)
DECIMAL(12.E-12)
DECIMAL(.3E-1)
DECIMAL(.456E-498)
DECIMAL(7.8E-35)
DECIMAL(9.01E-7985)
DECIMAL(234.567E-28)
NATURAL(1)
IDENTIFIER(E)
NATURAL(9)
IDENTIFIER(e)
DECIMAL(2.)
IDENTIFIER(e)
DECIMAL(3.)
IDENTIFIER(E)
Line 1, column 399: illegal character (.)
IDENTIFIER(e)
Line 1, column 402: illegal character (.)
IDENTIFIER(E)
SEMICOLON
